src/features/NavBar/NavBar.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useRef, useState } from "react";
+
+import "./NavBar.css";
+
+function NavBar() {
+  let navRef = useRef();
+  const [open, setOpen] = useState(false);
+  const toggleSettings = () => {
+    if (open) {
+      navRef.current.style.width = "0";
+      setOpen(false);
+    } else {
+      navRef.current.style.width = "20vw";
+      setOpen(true);
+    }
+  };
+  return (
+    <div className="navbar-container">
+      <Link to="/">
+        <img
+          className="logo"
+          src="https://media.giphy.com/media/FxfVpjpQb1kwKadK9L/giphy.gif"
+          alt="logo"
+        />
+      </Link>
+      <div className="navbar-contents">
```

Comment:
noice.. matches comp fine. are these working links now ? I know we dont have the pages but curious.. Ill play around with them

Reply:
The **Login** link works! The other links do successfully redirect the page but there isn't yet a component made/rendered to display for them.